src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write code here
+const wall = document.querySelector('.wall');
+const elementSpider = document.querySelector('.spider');
+
+const horizontal = (parseFloat(getComputedStyle(wall).width) / 2)
+  - (parseFloat(getComputedStyle(elementSpider).width) / 2);
+const vertical = (parseFloat(getComputedStyle(wall).height) / 2)
+  - (parseFloat(getComputedStyle(elementSpider).height) / 2);
+
+elementSpider.style.left = `${horizontal}px`;
+
+elementSpider.style.top = `${vertical}px`;
```

Comment:
You can make them 50% and then add transform: translate(-50%, -50%) to properly centre it. 
I guess this way is much more easier, but it's up to you
https://www.w3schools.com/css/tryit.asp?filename=trycss_align_transform